appgate_sdp/assets/monitors/monitor_SSH_Access_attempted.json

```diff
@@ -0,0 +1,34 @@
+{
+	"version": 2,
+	"created_at": "2025-04-02",
+	"last_updated_at": "2025-04-02",
+	"title": "SSH Access attempted",
+	"description": "An attempt has been made to access maintenance terminal",
+	"definition": {
+		"id": 168104482,
+		"name": "SSH Access attempted",
+		"type": "query alert",
+		"query": "sum(last_5m):sum:appgate.apn_audit_events.count{type:ssh_access_*} by {type,appliance_name,func,site_name,collective_name}.as_count() > 0",
+		"message": "Appliance: {{appliance_name.name}} \nAn attempt has been made to access maintenance terminal",
+		"tags": [
+			"agisdp_monitor"
```

Comment:
```suggestion
			"agisdp_monitor"
```
What is this for?

Reply:
This is a tag used by the Appgate team that they attach to their monitors.